### ISSUE TITLE
Death Crimson OX

### ISSUE THREAD

**Issue by animekai** [Rinnegatamante/flycast-vita-Compatibility#18]
# Game summary
- Game name: Death Crimson OX
- Game version:  US CDI

# flycast summary
- Version: v Pre-Release

# Issues

Game runs fine, is very playable, at 30fps most times, the pointer is super fast
# Screenshots
![2021-01-02-111642-165930](https://user-images.githubusercontent.com/32076097/103461492-cff70f80-4cec-11eb-9eb0-787bf7f65d62.png)
![2021-01-02-111803-902151](https://user-images.githubusercontent.com/32076097/103461493-d1283c80-4cec-11eb-86b8-fd4cc0e4c7ef.png)
![2021-01-02-111818-145242](https://user-images.githubusercontent.com/32076097/103461494-d1283c80-4cec-11eb-9658-5c9bd4f2313d.png)
![2021-01-02-111844-054088](https://user-images.githubusercontent.com/32076097/103461495-d1c0d300-4cec-11eb-8d47-3021f43d419d.png)
![2021-01-02-111857-768098](https://user-images.githubusercontent.com/32076097/103461496-d2596980-4cec-11eb-9d8f-3dfe977d2b10.png)
![2021-01-02-111914-151374](https://user-images.githubusercontent.com/32076097/103461497-d2f20000-4cec-11eb-9532-16acebbe6f38.png)
![2021-01-02-111930-317333](https://user-images.githubusercontent.com/32076097/103461499-d2f20000-4cec-11eb-9eca-14a7ea580c28.png)
![2021-01-02-111945-132211](https://user-images.githubusercontent.com/32076097/103461501-d38a9680-4cec-11eb-9a6c-33b4908656e9.png)

# Recommended labels

- A Playable
- B?
- C?

**Comment by Rinnegatamante** [comment]
Waiting for a confirmation the game is Playable to reopen this.

**Comment by animekai** [comment]
yes this game is playable, the framerate is acceptable for gameplay

**Comment by georgewaraw** [comment]
Runs great on 0.1. Slight visual glitches in the menu but nothing too intrusive (when toggling the options, you'll need to back out to the main menu and enter the options again to see the changes that you made). As for the pointer, the analog stick does make it move too fast but using the D-pad instead fixes this issue. The game controls surprisingly well with the D-pad too.
